### PR TITLE
Fixes Gatfruit

### LINKED ISSUE #24434: Gatfruits do not spawn revolvers
[Note]: # Server revision compiled on: 2017-02-24
10f38a4d1789a07401c35750ffeab39a43d9829a

Gatfruits do not seem to properly spawn a revolver when used in hand. Instead, they just disappear, leaving nothing behind. Tested with a player-grown gatfruit and an admin spawned one with the same result.

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
Freakin CYBERBOSS

fixes #24434

### CODE REVIEW

File: code/modules/hydroponics/grown.dm [hunk 1]
```diff
@@ -166,7 +166,6 @@
 	var/obj/item/T
 	if(trash)
 		T = generate_trash()
-	qdel(src)
-	if(trash)
 		user.put_in_hands(T)
```
Comment (ChangelingRain) [inline code/modules/hydroponics/grown.dm]: drop this item first so the new item is in the same hand

Comment (ExcessiveUseOfCobblestone) [inline code/modules/hydroponics/grown.dm]: @ChangelingRain which proc would I use for that, there's a couple of unequips now and I don't want to choose the wrong one.

Comment (Cyberboss) [inline code/modules/hydroponics/grown.dm]: Replace put_in_hands with `user.putItemFromInventoryInHandIfPossible(T, user.active_hand_index, TRUE)

Comment (Cyberboss) [inline code/modules/hydroponics/grown.dm]: You need to `qdel(src)` first, though, or the hand will still be occupied

**Comment by ExcessiveUseOfCobblestone** [comment]
@ChangelingRain @Cyberboss give this a quick check then it should be ready to set sail :3

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**